### DISCUSSION TITLE
Mining out the certain location

### DISCUSSION THREAD

**Discussion by DEATHToHacker** [PrismarineJS/mineflayer#1983] [Q&A]
Anyone know how to make bot mine out to certain cube? Ex. Mine a cube between -10 0 -10 and 10 64 10
If you know, give me a source code please

**Comment by amoraschi** [comment]
You can loop over the blocks, check if it's air, if it is, skip it, if it isn't, dig it

**Reply by DEATHToHacker** [reply]
I mean smth like, I need to mine out the entire chunk, but not mine in other chunk

**Reply by DEATHToHacker** [reply]
do you know how to do it?

**Reply by amoraschi** [reply]
What do you mean, just don't mine it if you don't want, you can configure the loop

**Reply by aesthetic0001** [reply]
@DEATHToHacker he just told you how to do it. You need to implement it yourself, unless u can find code that already accomplishes this.
